base/Base_compiler.jl

```diff
@@ -266,6 +266,21 @@ function strcat(x::String, y::String)
     return out
 end
 
+function file_exists(filename::String)
```

Comment:
This seems unrelated? Normally you'd call `access` with `F_OK` to implement this

Reply:
Eh, I put some commits on this branch by accident it seems..

Reply:
Fixed